packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -151,17 +156,21 @@ const start = async zcf => {
     return harden({ quoteAmount, quotePayment });
   };
 
+  // FIXME: We throw away the updater but shouldn't.
   const { notifier } = makeNotifierKit();
-  const zoe = zcf.getZoeService();
 
   /**
    * @typedef {Object} OracleRecord
    * @property {(timestamp: Timestamp) => Promise<void>=} querier
    * @property {number} lastSample
    */
 
-  /** @type {LegacyMap<Instance, Set<OracleRecord>>} */
-  const instanceToRecords = makeLegacyMap('oracleInstance');
+  /**
+   * @typedef {{}} OracleKey
```

Comment:
because TS is structurally typed, this make the type just documentation. Reading above it looks like
```suggestion
   * @typedef {Instance | Remotable<'fresh key'> } OracleKey
```

Reply:
It is the type of an object without properties, which is what I want to indicate.  Maybe this is where JSDoc types diverge from TS interfaces?

Reply:
> It is the type of an object without properties, which is what I want to indicate.

In that case, 👍 . I assumed an `OracleKey` was something more specific than an object without properties. 

> Maybe this is where JSDoc types diverge from TS interfaces?

I'm not aware of any divergences.